File: src/microservices/com.smi.microservices.ctpanonymiser/src/main/java/org/smi/ctpanonymiser/util/ExtractedFileStatus.java

package org.smi.ctpanonymiser.util;

public enum ExtractedFileStatus {
    /**
	 * Unused placeholder value
	 */
    Unused,

	/**
	 * The file has been anonymised successfully
	 */
	Anonymised,

	/**
	 * The file could not be anonymised and will not be retired
	 */
	ErrorWontRetry,

	/**
	 * The source file could not be found under the given filesystem root
	 */
	FileMissing,

	/**
	 * The source file was successfully copied to the destination
	 */
	Copied,
}

File: src/SmiServices/Common/Messages/Extraction/ExtractedFileStatus.cs
namespace SmiServices.Common.Messages.Extraction
{
    public enum ExtractedFileStatus
    {
        /// <summary>
        /// Unused placeholder value
        /// </summary>
        None = 0,

        /// <summary>
        /// The file has been anonymised successfully
        /// </summary>
        Anonymised,

        /// <summary>
        /// The file could not be anonymised and will not be retired
        /// </summary>
        ErrorWontRetry,

        /// <summary>
        /// The source file could not be found under the given filesystem root
        /// </summary>
        FileMissing,

        /// <summary>
        /// The source file was successfully copied to the destination
        /// </summary>
        Copied,
    }
}
